mmc-github/src/test/unit/java/info/novatec/metricscollector/github/MetricsRepositoryTest.java

```diff
@@ -30,37 +30,37 @@
     private InfluxService influxService;
 
     @MockBean
```

Comment:
For unit tests, we should use the Mockito implementation - either @Mock or Mockito.mock(), otherwise with @MockBean we say it is a mock and a bean which leads to Spring App context.